fabric-data-generation-api-v1/src/main/resources/fabric-data-generation-api-v1.accesswidener

```diff
@@ -39,6 +39,10 @@ transitive-accessible method net/minecraft/data/family/BlockFamilies register (L
 
 transitive-accessible field net/minecraft/client/data/BlockStateModelGenerator blockStateCollector Ljava/util/function/Consumer;
 transitive-accessible field net/minecraft/client/data/BlockStateModelGenerator modelCollector Ljava/util/function/BiConsumer;
+transitive-accessible field net/minecraft/client/data/BlockStateModelGenerator itemModelOutput Lnet/minecraft/client/data/ItemModelOutput;
+
+accessible field net/minecraft/client/data/ItemModelGenerator output Lnet/minecraft/client/data/ItemModelOutput;
+accessible field net/minecraft/client/data/ItemModelGenerator modelCollector Ljava/util/function/BiConsumer;
```

Comment:
I think these two lines are wrong? They should be `transitive-accessible`, right?

Reply:
Uhh, yes thats correct.